src/assets/css/simplify.css

```diff
@@ -0,0 +1,737 @@
+    position: absolute;
+    top: 1rem;
+    right: 1.2rem;
+    font-size: 1.7rem;
+    z-index : 100;
+}
+
+.navbar .pc_none{
+    display: none;
+}
+
+.navbar .mob_none{
+    display: flex;
+}
+
+@media only screen and (max-width: 768px) {
+    .container-xl, .container-md{
+        padding: 0 1rem;
+    }
+    .navbar .pc_none{
+        display: flex;
+    }
+    
+    .navbar .mob_none{
+        display: none;
+    }
+}
+
+.navbar .collapse-nav-btn button {
+    z-index: 100;
+    border: none;
+    background: none;
+    position: absolute;
+    top : 1.5rem;
+    right: 1rem;
+    font-size: 1.3rem;
+}
+
+.navbar .menuNav {
+    min-height: 100vh;
+    position: absolute;
+    top:0;
+    width:100%;
+    padding: 1rem;
+    background: var(--color-white);
+    z-index: 10;
+    box-shadow: 0 5px 15px rgb(0 0 0 / 8%);
+    right: -100%;
+    transition: 0.5s;   
+}
+
+.menuNav .menuNav-list .menuNav-list-name {
+    font-size: 1.5rem;
+    font-weight: 500;
+    padding: 0.75rem 0;
+}
+
+.menuNav .menuNav-list .menuNav-item {
+    padding: 0.75rem 0.5rem;
+    font-size: 1.2rem;
+    font-weight: 400;
+
+}
+.navbar-collapse > .navbar-nav {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+}
+
+.navbar-nav > .nav-item {
+    font-size: 1.25rem;
+    padding: 0 0.5rem;
+    margin-right: 1rem;
+}
+
+.side-bar {
+    margin-left: 3rem;
+    padding: 1rem;
+}
+
+.side-bar > .sidebar-list {
+    margin-bottom: 1rem;
+}
+
+.sidebar-list > .sidebar-list-name {
+    font-size : 1.25rem;
+    font-weight: 500;
+    color: var(--color-primary-500);
+    text-transform: uppercase;
+}
+
+
+.sidebar-list .sidebar-items {
+    padding : 0.25rem;
+}
+
+.sidebar-items .sidebar-item {
+    padding: 0.75rem 0;
+    color: var(--color-primary-300);
+}
+
+
+.sidebar-items > .sidebar-item:hover{
+    border-bottom: 0.15rem solid var(--color-primary-500);
+    transition: 0.1s ease;
+    transition-property: border-bottom;
+    font-weight: 600;
+}
+
+@media only screen and (max-width : 768px) {
+    .side-bar{
+        display: none;
+    }
+}
+
+.component-box {
+    margin: 1rem auto;
+    padding: 0.5rem;
+    border-radius: 7px;
+    box-shadow: 0 0 2.5px rgb(0 0 0 / 12%);
+}
+
+.list {
+    border-radius: 10px;
+    box-shadow: 0 3px 16px 0 rgb(0 0 0 / 11%);
+}
+
+.list > ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.list > ul > li {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem;
+    width: 100%;
+    border-bottom: 1px solid var(--color-primary-200);
+}
+
+.list > ul > li:last-child {
+    border-bottom: none;
+}
+
+.list.list-striped li {
+    padding: 1rem;
+    border: none;
+}
+
+.list.list-striped li:nth-child(2n) {
+    background: var(--color-primary-200);
+}
+
+.modal {
+    display: none;
+    position: absolute;
+    margin-left: auto;
+    margin-right: auto;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 100;
```

Comment:
you can use inset shorthand property with position: absolute.  something i discovered today

https://developer.mozilla.org/en-US/docs/Web/CSS/inset

Reply:
Done. Please Review.